package.json

```diff
@@ -147,7 +147,7 @@
     "electron-notarize": "^1.0.0",
     "electron-reloader": "^1.2.0",
     "eslint": "^7.6.0",
-    "eslint-config-expensify": "^2.0.18",
+    "eslint-config-expensify": "git+https://github.com/Expensify/eslint-config-expensify.git#afc71e84361cc68af4a85b425b3a599c4ac1861d",
```

Comment:
I thought this would be referencing the commit in your PR: https://github.com/Expensify/eslint-config-expensify/pull/37/commits/e1edb2cfe7fd91b9b131187cb19481ba9d29705b

But `afc71e84361cc68af4a85b425b3a599c4ac1861d` is a different hash, is this correct?

```
App git:(main) ✗ npm install
npm ERR! code 128
npm ERR! Command failed: git checkout afc71e84361cc68af4a85b425b3a599c4ac1861d
npm ERR! fatal: reference is not a tree: afc71e84361cc68af4a85b425b3a599c4ac1861d
```

Reply:
Yeah it needs to be updated to use the actual version bump. But we need to bump the version first.

I'm not sure what is up with those errors they just started happening randomly and breaking everything - so maybe updating to the latest commit will help.